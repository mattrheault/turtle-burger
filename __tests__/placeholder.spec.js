describe('universal truth', () => {
  it('should be true', () => {
    expect(true).toMatchSnapshot();
  });
});
