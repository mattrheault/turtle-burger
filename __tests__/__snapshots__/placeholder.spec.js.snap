// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`universal truth should be true 1`] = `true`;
